logrotation/README.md

```diff
@@ -0,0 +1,34 @@
+# Enable Log Rotation for Components
+
+Log rotation options for components
+  - linux cron job
+  - logrotate
+
+## Linux Cron Job
+  - At the entry point,
+    - Run a startup script.\
+    Input to the script is config url.\
+    The script will add a crontab entry to periodically schedule the logrotate script.
+    - Start the crond service.
+    - Run the component entrypoint script.
```

Comment:
how this entrypoint script is different from startup script (line 9) ?

Reply:
rephrased it,  please check. I meant to continue the entry point operations that each component has.